tests/checkpoint/cloud_config.py

```diff
@@ -56,11 +56,13 @@ def store_set(self, key, value, **kwargs):
 
 def list_keys(self, prefix: Tuple = ()):
     if self.ge_cloud_resource_type == GXCloudRESTResource.EXPECTATION_SUITE:
-        return (
-            GXCloudRESTResource.EXPECTATION_SUITE,
-            "1212e79d-f751-4c6e-921d-26de2b1db174",
-            "suite_name",
-        )
+        return [
+            (
+                GXCloudRESTResource.EXPECTATION_SUITE,
+                "1212e79d-f751-4c6e-921d-26de2b1db174",
```

Comment:
@Shinnnyshinshin was this key generated by GX-Cloud for the test?  Thanks.

Reply:
Hi @alexsherstinsky great question. It's actually part of a fixture that is used in a Mock: 

https://github.com/great-expectations/great_expectations/blob/184144211940df099c440b2eb5e9d0c4691ea3ce/tests/checkpoint/test_checkpoint.py#L4799